projects/ADuCM360_demo_cn0338/src/Cmd.cpp

```diff
@@ -222,6 +222,10 @@ void Cmd_ReadData(void)
       char c = uart_rx_queue[uart_rx_head];
       if (c >= ' ' && c <= '~') {
          handle_ascii_data(c);
+         if(uart_rx_head == 63)
```

Comment:
Do you have a define for RING_BUFFER_LEN ? I would use that. If you encrease the size of the buffer you will have to update here manually also.

Reply:
I think this is the define to use UART_RX_QUEUE_SIZE